datadog_checks_base/tests/test_http.py

```diff
@@ -1151,6 +1151,6 @@ def test_delete_session_option_override(self):
 
 class TestIntegration:
     def test_session_timeout(self):
-        http = RequestsWrapper({'persist_connections': True}, {'timeout': 0.300})
+        http = RequestsWrapper({'persist_connections': True}, {'timeout': 0.001})
         with pytest.raises(requests.exceptions.Timeout):
-            http.get('https://httpstat.us/200?sleep=500')
+            http.get('https://httpbin.org/get')
```

Comment:
Note that `httpbin.org` has a `delay` endpoint, which might reflect better what we want (here we're testing the timeout based on an assumption on latency):

```suggestion
            http.get('https://httpbin.org/delay/0.1')
```

Reply:
Nice find, I thought they didn't had this feature.